client.go

```diff
@@ -271,6 +271,21 @@ func (uc Client) GetFeaturesByPattern(pattern string) []api.Feature {
 	}
 	return result
 }
+// GetFeaturesByStrategy retrieves all features whose used a given Strategy
+func (uc Client) GetFeaturesByStrategy(strategyName string) []api.Feature {
+	result := make([]api.Feature, 0)
+	features := uc.repository.getAllToggles()
+	for _, feat := range features {
+		for _, s := range feat.Strategies {
+			foundStrategy := uc.getStrategy(s.Name)
```

Comment:
I'm not sure we need to fetch the strategy by its name to then compare its name.